examples/authorization_details_mso_doc.json

```diff
@@ -3,14 +3,14 @@
     "type": "openid_credential",
     "credential_configuration_id": "org.iso.18013.5.1.mDL",
     "claims": {
-      "org.iso.18013.5.1": {
-        "given_name": {},
-        "family_name": {},
-        "birth_date": {}
-      },
-      "org.iso.18013.5.1.aamva": {
-        "organ_donor": {}
-      }
+      "org.iso.18013.5.1": [
+        {"path": ["given_name"]},
+        {"path": ["family_name"]},
+        {"path": ["birth_date"]}
+      ],
+      "org.iso.18013.5.1.aamva": [
+        {"path": ["organ_donor"]}
+      ]
```

Comment:
This could be changed to use the same structure as in the new query language (namespace/claim_name)